test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+set -a
```

Comment:
`set -a` puts all variables you set into the environment, so you don't have to keep typing `export`.